src/__test__/components/data-exploration/differential-expression-tool/DiffExprResults.test.jsx

```diff
@@ -336,4 +336,20 @@ describe('DiffExprResults', () => {
     // Expect table to contain Empty component
     expect(empty.length).toEqual(1);
   });
+
+  it('Advanced filter button opens and closes the modal', () => {
```

Comment:
We could also have a test for checking that the advanced filter button doesn't show up when diff expr results aren't being shown

Reply:
ahh you're right, ok